src/Bridge/Doctrine/Purger/OrmPurger.php

```diff
@@ -79,9 +79,12 @@ public function create(PurgeMode $mode, PurgerInterface $purger = null): PurgerI
 
     /**
      * @inheritdoc
+     * @throws \Doctrine\DBAL\DBALException
      */
     public function purge()
     {
+        $this->manager->getConnection()->prepare('SET FOREIGN_KEY_CHECKS = 0;')->execute();
```

Comment:
But I don't need cascade delete, therefore I can't execute command with  --purge-with-truncate option because of foreign keys... How I can do this?